var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,29 +35,55 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
 
-    version('develop')
-    version('2.1', tag='2.1')
-    version('2.0.3', tag='2.0.3')
+    version('3.0_beta', git='https://bitbucket.org/scheres/relion-3.0_beta.git')
+    version('2.1', git='https://github.com/3dem/relion.git', preferred='true', tag='2.1')
+    version('2.0.3', git='https://github.com/3dem/relion.git', tag='2.0.3')
+    version('develop', git='https://github.com/3dem/relion.git')
```

Comment:
These version-specific git URLs can be removed unless they are different than the package-level git.

Reply:
Fixing now!